webview-ui/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
💡 Comments please.

Reply:
As mentioned somewhere else, you can't add comments to this file, but I think everything is covered in `/docs/package-scripts` and `/docs/webview-development`.